project/plugins.sbt

```diff
@@ -24,3 +25,5 @@ addSbtPlugin("com.timushev.sbt" % "sbt-updates" % "0.1.10")
 addSbtPlugin("com.gu" % "sbt-riffraff-artifact" % "0.8.1")
 
 addSbtPlugin("com.typesafe.sbt" % "sbt-git" % "0.8.5")
+
+addSbtPlugin("org.jetbrains.teamcity.plugins" % "sbt-teamcity-logger" % "0.3.0")
```

Comment:
I think that this was the plugin caused the following problem: https://github.com/JetBrains/sbt-tc-logger/issues/9